### DISCUSSION TITLE
How can I find which photos were not included in the library quickly?

### DISCUSSION THREAD

**Discussion by ricky0829** [photoprism/photoprism#2266] [Q&A]
At first I want to say thanks! Thanks all of develpers, photoprism is really amazing!
My question below:
I have 1208 photos (including 60 videos) in my original album. 
![image](https://user-images.githubusercontent.com/28993319/163982840-c54496ec-50dd-4c7f-8ba6-62a852255c62.png)
Actually I have much more but I have to figure this question out before I import all my photos into it.
After index them, I got 1191 photos (including 60 videos) in the library, which means **I lost 17 photos.**
![image](https://user-images.githubusercontent.com/28993319/163982906-d257e89e-a264-4466-aac1-cf694d051229.png)

I tried re-index, truned Quality Filter off and re-index, both didn't work. And I tried to search photo by its original file name, it cannot be searched. 
So I want to know how can I find which photos were not included in the library quickly? Or is there any way to import the missing 17 photos into the library?

**Comment by lastzero** [comment]
Could be stacked, like when you have multiple files that belong together.

https://docs.photoprism.app/getting-started/troubleshooting/#missing-pictures

**Reply by graciousgrey** [reply]
Your library contains files, whereas the Search counts unique photos and videos.

Photos and videos may have more than one file, for example:

- A raw file + related jpg file + related xmp file = 3 files, 1 photo
- A mp4 file + related jpg file = 2 files, 1 video

It is also possible that multiple .jpg files are stacked because they are related to each other

**Reply by ricky0829** [reply]
Thanks for you two. I clicked the original below the library, and the number looks correct. I'll check it later.
